pio/ppm/ppm.c

```diff
@@ -8,10 +8,10 @@
  * 8-channel Radio Control PPM example
  * Output on GPIO3
  *
- *  >=5ms       0.5ms    1-2ms       0.5ms     1-2ms         0.5ms    1-2ms       0.5ms
- *  synchro     strobe   ch0 value   strobe    ch1 value     strobe   ch8 value   strobe
- *              _______               _______                _______              _______
- * _______...__|       |_____________|       |___________...|       |____________|       |_
+
+ *  synchro     /  ch0 value 1-2ms  \/  ch0 value 1-2ms  \   /  ch8 value 1-2ms  \
+ *  >=5ms       _______    -500ms     _______                 _______              _______
+ * _______...__| 0.5ms |_____________| 0.5ms |___________|...| 0.5ms |____________| 0.5ms |_
```

Comment:
I don't have hardware to test this on, but is there definitely a 0.5ms pulse after the ch8 value, or does it just go straight into the next 5ms synchro pulse?
Also, I suspect the "-500ms" should probably have been "-500us"?

Reply:
My friend tested that with a real flight controller, ch8 worked.
There is a pulse after ch8, it's drawn. 
"500ms" should not be here at all.
Excuse me for being so inaccurate.
I have changed the comment once again, hopefully it's better now